modules/user_picachoo.py

```diff
@@ -0,0 +1,65 @@
+import re
+import requests
+import json
+
+from modules.utils.data import prepare_binary_from_url
+
+def main(bot, *args, **kwargs):
+    """
+    picachoo
+    Random pic from picachoo.ru Usage: /picachoo [page number from 0] [pic number from 0]
+    """
+    user_page = None
+    user_index = None
+
+    try:
+        user_page = int(args[0])
+        user_index = int(args[1])
+    except:
+        pass
+
+    matricies = json.loads(bot.store.get('picachoo_matricies') or '[]')
+    curr_index = int(bot.store.get('picachoo_index') or 0)
+    curr_page = int(bot.store.get('picachoo_page') or 0)
+
+    if not user_page is None:
+        curr_page = user_page
+
+    if curr_index >= len(matricies) or not user_page is None:
+        response = requests.get('https://www.picachoo.ru/more/' + str(curr_page + 1), verify=False)
```

Comment:
Use `format` instead of str concatenation

(`'https://www.picachoo.ru/more/{}'.format(curr_page + 1)`)